daphne_worker_test/helper.env

```diff
@@ -1,5 +1,6 @@
 DAP_DEPLOYMENT = "prod"
 DAP_AGGREGATOR_ROLE = "helper"
+DISABLE_JOB_QUEUE_FINISH = "true"
```

Comment:
1. The convention is to prefix variables with "DAP_" that impact Daphne-Worker's behavior.
2. Maybe we should give it a name that points to the fact that this hacks around a bug. Maybe `DAP_ISSUE73_DISABLE_AGG_JOB_QUEUE_GARBAGE_COLLECTION`. (being overly verbose is an asset here, I think.)

Reply:
Good idea, I went with your suggestion of `DAP_ISSUE73_DISABLE_AGG_JOB_QUEUE_GARBAGE_COLLECTION`.